packages/massa-web3/src/experimental/client/interfaces.ts

```diff
@@ -33,6 +61,7 @@ export interface BlockchainClient {
   getEvents(filter: EventFilter): Promise<SCOutputEvent[]>
   getChainId(): Promise<bigint>
   getMinimalFee(): Promise<bigint>
+  executeReadOnlyCall(opt: ReadOnlyCallParams): Promise<ReadOnlyCallResult>
```

Comment:
The parameter name is misleading; "opt" stands for optional. Here, nothing is optional.